docs/release-notes/next-fortuna/PT-162539245-monitoring

```diff
@@ -0,0 +1 @@
+* Add `aemon` application to handle network monitoring.
```

Comment:
Could this be made more user oriented? E.g. "* Adds network monitoring capability to the node. This is disabled by default. Please refer to the `docs/monitoring.md` document for the intended usage."